add(platform): limited and early availability tags (#29)
aiven/aiven-docs: docs/products/cassandra/howto/manage-cross-cluster-replication.md

---
title: Manage cross-cluster replication in Aiven for Apache Cassandra®
limited: true
---

Learn how to update Apache Cassandra® services that has cross-cluster replication (CCR) enabled: change the service plan and add an extra disk space. Find out how to set up the replication factor and the consistency level for your CCR-enabled Apache Cassandra® services.

## Prerequisites

> CCR is a [limited availability feature](/docs/platform/concepts/beta_services). To try it
> out, contact the sales team at [sales@aiven.io](mailto:sales@aiven.io).

Reviewer: Let's talk about harmonizing these "contact sales" on Tuesday. In the mean time, let's do shorter and more resusable perhaps, to already lay ground work for uniformity?

```suggestion
This feature is in limited availability. Contact the sales team(mailto:sales@aiven.io) to enable it.
```
